frontend/src/app/testResults/uploads/Uploads.tsx

```diff
@@ -115,21 +124,41 @@ const Uploads = () => {
         setErrorMessageText(
           "There was a server error. Your file has not been accepted."
         );
+        appInsights?.trackEvent({
+          name: "Spreadsheet upload server error",
+          properties: {
+            org: orgName,
+            user: user?.email,
+          },
+        });
       } else {
         const response = await res.json();
 
         if (response?.reportId) {
           setReportId(response?.reportId);
+          appInsights?.trackEvent({
+            name: "Spreadsheet upload success",
+            properties: {
+              "report ID": response.reportId,
```

Comment:
can we change this key to `reportId`?

Reply:
actually the reason I did it this way and not camel case is because of [this conversation](https://github.com/CDCgov/prime-simplereport/pull/4305#discussion_r984682647). seems like we should try to be consistent even though they are different events